config/environments/development.rb

```diff
@@ -115,6 +115,11 @@
   REPOSITORY_STORAGE = "#{::Rails.root.to_s}/data/dev/repos"
 
   ###################################################################
+  # Directory where Repositories will be created. Make sure MarkUs is allowed
+  # to write to this directory
+  REPOSITORY_STORAGE_GITOLITE = "#{::Rails.root.to_s}/data/dev/repos/gitoliteRepos"
```

Comment:
Oh, I was referring more to the line length for the readability part. I assumed the "to_s" thing was fine since the other constants were like that.